client/my-sites/plan-features/index.jsx

```diff
@@ -88,27 +93,31 @@ PlanFeatures.propTypes = {
 	available: PropTypes.bool,
 	onUpgradeClick: PropTypes.func,
 	// either you specify the plan prop or isPlaceholder prop
-	plan: React.PropTypes.string,
-	isPlaceholder: PropTypes.bool
+	plan: PropTypes.string,
+	isPlaceholder: PropTypes.bool,
+	isInSignup: PropTypes.bool
```

Comment:
It looked like `isInSignup` was being used to determine which data to use, eg from `QuerySitePlans` ( `/sites/$siteId/plans`) or `QueryPlans` (`/plans`). 

A future use case might be that we add another static marketing page, and so the `isInSignup` prop wouldn't make much sense.